R/data_marium.R

```diff
@@ -0,0 +1,40 @@
+#' What America’s Governors Are Talking About
+#' 
+#' The raw data behind the story
+#' "What America’s Governors Are Talking About"
+#' \url{https://fivethirtyeight.com/features/what-americas-governors-are-talking-about/}
+#' 
+#' @format A data frame with 50 rows representing the 50 U.S. states and 5 variables:
+#' \describe{
+#'   \item{state}{the state}
+#'   \item{governer}{the name of the state's governer}
+#'   \item{party}{the party of the state's governer}
+#'   \item{filename}{the filename of the speech in the speeches folder}
+#'   \item{url}{a link to an official/media source for the speech}
+#'   }
+"index"
+
+
+
+#' What America’s Governors Are Talking About
```

Comment:
Can @mariumtapal please use the `@rdname` directive to document these two data sets in the same article?

Reply:
Please also add yourself as a contributor in the `DESCRIPTION` file and the `NEWS.md`